examples/apps/color_finder.py

```diff
@@ -0,0 +1,740 @@
+
+        text = cozmo.annotate.ImageText('Looking for {}'.format(self.color_to_find), color = 'white')
+        text.render(d, (0, 0, image.width, image.height))
+
+    def on_cube_tap(self, evt, obj, **kwargs):
+        '''The blinking white cube switches the viewer between normal mode and pixel mode.
+        The other illuminated cube toggles self.color_to_find.
+        
+        '''    
+        if obj.object_id == self.color_selector_cube.object_id:
+            self.toggle_color_to_find()
+        elif obj.object_id == self.grid_cube.object_id:
+            self.robot.world.image_annotator.annotation_enabled = not self.robot.world.image_annotator.annotation_enabled
+
+    def toggle_color_to_find(self):
+        '''Sets self.color_to_find to the next color in self.possible_colors_to_find.'''    
+        self.color_to_find_index += 1
+        if self.color_to_find_index == len(self.possible_colors_to_find):
+            self.color_to_find_index = 0
+        self.color_to_find = self.possible_colors_to_find[self.color_to_find_index]
+        self.color_selector_cube.set_lights(map_color_to_light[self.color_to_find])
+
+    async def on_new_camera_image(self, evt, **kwargs):
+        '''Processes the blobs in Cozmo's view, and determines the correct reaction.'''
+        downsized_image = self.get_low_res_view()
+        self.update_pixel_matrix(downsized_image)
+        blob_detector = BlobDetector(self.pixel_matrix, self.possible_colors_to_find)
+        blob_info = blob_detector.get_center_of_blob_with_color(self.color_to_find)
+        if blob_info:
+            if self.state == LOOK_AROUND_STATE:
+                self.state = FOUND_COLOR_STATE
+                if self.look_around_behavior:
+                    self.look_around_behavior.stop()
+            self.on_finding_a_blob(blob_info)
+        else:
+            self.robot.set_backpack_lights_off()
+            self.state = LOOK_AROUND_STATE
+
+    def update_pixel_matrix(self, downsized_image):
+        '''Updates self.pixel_matrix with the colors from the current camera view.
+
+        Args:
+            downsized_image (PIL image): the low-resolution version of self.robot.world.latest_image
+
+        '''
+        for i in range(self.pixel_matrix.num_cols):
+            for j in range(self.pixel_matrix.num_rows):
+                r, g, b = downsized_image.getpixel((i, j))
+                self.pixel_matrix.at(i, j).set(self.approximate_color_of_pixel(r, g, b))
+        self.pixel_matrix.fill_gaps()
+
+    def approximate_color_of_pixel(self, r, g, b):
+        '''Returns the approximated color of the RGB value of a pixel.
+
+        Args:
+            r (int): the amount of red in the pixel
+            g (int): the amount of green in the pixel
+            b (int): the amount of blue in the pixel
+
+        Returns:
+            closest_color (string): the name of the color closest to the input RGB color
```

Comment:
Returns should not name a var name. Should instead be like:
Returns:
string that specifies the name of the color closest to the input RBG color.

Applies to all Returns in the file.